### ISSUE TITLE
Package request: remind

### ISSUE THREAD

**Issue by AnderRasoVazquez** [termux/termux-packages#75]
[**Remind**](https://www.roaringpenguin.com/products/remind) is a sophisticated calendar and alarm program. It includes the following features:

+ A sophisticated scripting language and intelligent handling of exceptions and holidays.
+ Plain-text, PostScript and HTML output.
+ Timed reminders and pop-up alarms.
+ A friendly graphical front-end for people who don't want to learn the scripting language.
+ Facilities for both the Gregorian and Hebrew calendars.
+ Support for 12 different languages.

**Basically is a plain text calendar for the terminal**. Really great for those who prefer to host their own data instead of using Google or other services. Keep up the good work! =)

**Comment by fornwall** [comment]
Seems like a nice tool that's small and easy to package! It's been added, install with

    apt install remind

and create new issues if you run into any problem!
